docker-compose.yml

```diff
@@ -386,3 +392,47 @@ services:
         source: ${HOME}/.ivy2
         target: /root/.ivy2
         read_only: false
+  otel-collector:
+    profiles:
+      - tracing
+    image: otel/opentelemetry-collector:0.86.0
+    command: [ "--config=/etc/otel-collector.yaml" ]
+    volumes:
+      - ./otel-collector.yaml:/etc/otel-collector.yaml
+  # Tempo runs as user 10001, and docker compose creates the volume as root.
```

Comment:
Thank you for the comment, I got tripped up at first because I know Minio uses port 10001.